### DISCUSSION TITLE
removing an item from scalar list

### DISCUSSION THREAD

**Discussion by kevinmachstudio** [prisma/prisma#8959] [Q&A] [answered]
in Prisma v2.2.0, the ability to add an item to scalar list was implemented https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-scalar-lists-arrays#adding-items-to-a-scalar-list, is there also an ability to remove an item from scalar list?

If not, are there any workarounds?

**Comment by TasinIshmam** [comment, marked as answer]
Hi, sorry for the late response. 

As of writing, there's no method to splice/remove items from a scalar list using Prisma. You would have to fetch the scalar list from your database, modify it manually in your application code and overwrite the record in your database with an ```update``` operation. 

There is a [feature request for this](https://github.com/prisma/prisma/issues/7401), please feel free to follow/comment with your use-case to help us track demand for this feature.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
